tutorials-teaming-in-litmus/index.html

```diff
@@ -106,6 +106,11 @@ <h2 is-upgraded>Editing an invitation</h2>
 <p>In an event that you want to revoke a sent invitation, or edit the roles set for a specific user when the initial invitation was sent, the portal allows you to edit the invitation already sent to users. </p>
 <p>Navigate to the invited tab, here you would be able to change the role of an invited user or delete the user from the list of invitees by clicking the basket button. </p>
 <p class="image-container"><img style="width: 624.00px" src="img/38747f44089d9bfb.png"></p>
+<h2 is-upgraded>Accept an invite</h2>
```

Comment:
Can we update the heading to **"Accepting an invitation"**? This will keep it in sync with the previous heading.